packages/react-components/src/accordion/src/AccordionItem.tsx

```diff
@@ -1,15 +1,28 @@
 import "./Accordion.css";
 
+import { AccordionBuilderHeaderProps, AccordionBuilderPanelProps } from "./useAccordionItems";
 import { AccordionHeader } from "./AccordionHeader";
 import { AccordionPanel } from "./AccordionPanel";
 import { Disclosure } from "../../disclosure";
 import { mergeProps, useEventCallback } from "../../shared";
 import { useAccordionContext } from "./AccordionContext";
 
+export interface AccordionItemProps {
+    /**
+     * @ignore
+     */
+    item: {
```

Comment:
Looking at the interface, we can wander why those 4 property are not directly at the root. Why the item indirection?